### PR TITLE
[RELAY]Testing Inception, Squeezenet, VGG port

### PR DESCRIPTION
**Pull request by siju-samuel**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by siju-samuel** [comment]
@eqy @Huyuwei @slyubomirsky please help to review. TIA

**Comment by siju-samuel** [comment]
@eqy @Huyuwei @slyubomirsky @tqchen  please help to review.

File: python/tvm/relay/testing/inception_v3.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+def Conv(data, num_filter, kernel=(1, 1), stride=(1, 1), pad=(0, 0), name=None, suffix=''):
+    conv = layers.conv2d(
+        data=data,
+        channels=int(num_filter),
+        kernel_size=kernel,
+        strides=stride,
+        padding=pad,
+        name='%s%s_conv1' % (name, suffix))
+
+    #bn = sym.batch_norm(data=conv, name='%s%s_batchnorm' % (name, suffix), epsilon=2e-5)
```
Comment (yzhliu) [inline python/tvm/relay/testing/inception_v3.py]: remove the comment, so as those below.

**Comment by tqchen** [comment]
@yzhliu can you do a follow-up review and merge it if it is OK?

**Comment by tqchen** [comment]
Thanks, @siju-samuel I merged this in. However, the current code still has problems that need to be fixed, see https://github.com/dmlc/tvm/pull/2163 on followups

**Pull request merged**